src/Components/AdminPage/Login/loginUser.js

```diff
@@ -20,13 +20,24 @@ class Login extends Component {
       headers: {
         'Content-Type': 'application/json'
       },
-      body: JSON.stringify(this.state.login)
+         body: JSON.stringify(this.state.login)
+    }) 
+    .then(response => {
+      this.handleErros(response.ok)
+       return response.json()
     })
-      .then(response => response.json())
-      .then(console.log)
-      .catch(this.setState({ error: true }))
+    .then(console.log)
+    .catch(error => alert(error.message))
   }
 
+  handleErros = (validation) => {
+    if(validation){
```

Comment:
esto podria ser ```this.setState({error: !validation})```?

Reply:
si si funciona, pero no en el else

Reply:
lo voy a implemetar asi

Reply:
Ya en este commit lo arregle como sugieres y funciona muy bien. 

Igual quiero aclarar que este pull request era solo para acomodar el formulario que envia el contenido a la pagina principal ya que fue una tarea que le tomo bastante tiempo al equipo completar se decidio hacerle un pullR solo para eso, el manejo de los errores y el feedback del usuario esta en la respectiva rama con su pullR **change way of handleErrors** donde tomado la sugerencias de aqui lo voy a aplicar alla ya que comparte algunas cosas